### ISSUE TITLE
Document entryPoint and idls options in as-thrift docs

### ISSUE THREAD

**Issue by malandrew** [uber/tchannel-node#319]
The `entryPoint` option needs to be documented in the as-thrift documentation. Specifically, we need to call out that relative paths only work with the entryPoint documents. 

We should also document the `idls` option that allows the user to read all the sources themselves and provide them in an object where the keys represent the relative path.

Lastly, it might be worth having a better error when trying to load a file with the `source` option when the source itself contains relative includes.

**Comment by rajeshsegu** [comment]
We have library where we feed in the source of the idl file, but all the relative includes fail. I think `entryPoint` attribute could be useful here. I will try to add documentation in case you guys are ok.

**Comment by malandrew** [comment]
@rajeshsegu yes, a pull request with documentation would be much appreciated.
